src/pages/expenses/create/Create.tsx

```diff
@@ -26,42 +26,53 @@ import { TaxSettings } from './components/Taxes';
 import { ValidationBag } from 'common/interfaces/validation-bag';
 import { AxiosError } from 'axios';
 import { route } from 'common/helpers/route';
+import { useAtom } from 'jotai';
+import { expenseAtom } from '../common/atoms';
+import { useHandleChange } from '../common/hooks';
 
 export function Create() {
   const [t] = useTranslation();
 
   const navigate = useNavigate();
 
   const { documentTitle } = useTitle('new_expense');
+
   const { data } = useBlankExpenseQuery();
 
+  let mounted = false;
```

Comment:
Generally you should avoid this style of variables, as they aren't reactive (won't re-render).

Reply:
As we discussed on the Slack this part is removed from technical reasons